recipes/r-gmp/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '0.5-13.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-gmp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: gmp_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/gmp_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/gmp/gmp_{{ version }}.tar.gz
+  sha256: 2f805374a26742cd43f6b2054130d8670eda1940070aabb9971e9e48226d0976
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - gmp >=5.0.1,<7
```

Comment:
But this will only use gmp for windows builds, we also need it for Linux, isn't it?